ts/features/itwallet/navigation/useItwLinkingOptions.tsx

```diff
@@ -28,6 +29,9 @@ export const useItwLinkingOptions = (): PathConfigMap<AppParamsList> => {
         }),
         ...(canItwBeActivated && {
           [ITW_ROUTES.DISCOVERY.INFO]: "discovery/info"
+        }),
+        ...(itwAlreadyActive && {
+          [ITW_ROUTES.DISCOVERY.ALREADY_ACTIVE_SCREEN]: "discovery/info"
```

Comment:
```suggestion
        ...(canItwBeActivated && {
          [ITW_ROUTES.DISCOVERY.INFO]: "discovery/info"
        }),
        ...(itwAlreadyActive && {
          [ITW_ROUTES.DISCOVERY.ALREADY_ACTIVE_SCREEN]: "discovery/info"
```

These two routes should be mutually exclusive. What do you think?

Reply:
@mastro993 Maybe we could do something like this?

```
  ...(canItwBeActivated
    ? { [ITW_ROUTES.DISCOVERY.INFO]: "discovery/info" }
    : itwAlreadyActive
    ? { [ITW_ROUTES.DISCOVERY.ALREADY_ACTIVE_SCREEN]: "discovery/info" }
    : {})
```

Reply:
Yes, but inverted!

Reply:
How about something like this?

```ts
{
  ...(isUserAllowedToItw && {
    [ITW_ROUTES.ISSUANCE.CREDENTIAL_ASYNC_FLOW_CONTINUATION]:
      "credential/issuance",
    [isItwValid
      ? ITW_ROUTES.DISCOVERY.ALREADY_ACTIVE_SCREEN
      : ITW_ROUTES.DISCOVERY.INFO]: "discovery/info"
  })
}
```